release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1738,6 +1738,11 @@ There is no workaround for this issue. (link:https://issues.redhat.com/browse/OC
 
 [id="ocp-telco-ran-4-19-known-issues_{context}"]
 
+* In the event of a crash, the `mlx5_core` NIC driver causes an OOM issue and `kdump` does not save the vmcore file in `/var/crash`.
```

Comment:
```suggestion
* In the event of a crash, the `mlx5_core` NIC driver causes an out-of-memory issue and `kdump` does not save the `vmcore` file in `/var/crash`.
```
eh, be easier to read without the OOM acronym for a user
`vmcore` backticked below, missed backticks here